scripts/network/networkCONF.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+vpn=$1
+setupVars="/etc/pivpn/${vpn}/setupVars.conf"
+
+if [ ! -f "${setupVars}" ]; then
+	echo "::: Missing setup vars file!"
+	exit 1
+fi
+
+# shellcheck disable=SC1090
+source "${setupVars}"
+
+scriptDir="/opt/pivpn"
+
+# shellcheck disable=SC2034
+loadFirewallRulesArrays(){
```

Comment:
I know it's still WIP but I think that this section should check if IPv6 is enabled or disabled. E.g. a IPv6 device is not defined when the IPv6 ping check fails. Hence some of the following commands will fail.

Reply:
Thanks, I will fix this. It doesn't actually cause problems because all those boring functions in the `*FUNC.sh` files do check for IPv6, but yes, some of these arrays `loadFirewallRulesArrays` have undefined variables now.